twitter_ads/enum.py

```diff
@@ -96,6 +76,23 @@ def enum(**enums):
     OTHER='OTHER'
 )
 
+GOAL = enum(
+    APP_CLICKS='APP_CLICKS',
+    APP_INSTALLS='APP_INSTALLS',
+    ENGAGEMENT='ENGAGEMENT',
+    FOLLOWERS='FOLLOWERS',
+    LINK_CLICKS='LINK_CLICKS',
+    MAX_REACH='MAX_REACH',
+    PREROLL='PREROLL',
+    PREROLL_STARTS='PREROLL_STARTS',
+    REACH_WITH_ENGAGEMENT='REACH_WITH_ENGAGEMENT',
+    VIDEO_VIEW='VIDEO_VIEW',
+    VIEW_3S_100PCT='VIEW_3S_100PCT',
```

Comment:
Didn't realize these are valid goals too (`VIEW_3S_100PCT`, `VIEW_6S` and `VIEW_15S`). These should be `pay_by` if I'm not mistaken

Reply:
Thanks for the callout. Let me double check on this.

Reply:
Confirmed that `VIEW_3S_100PCT` works when creating a line item. Also checked the field values internally. Thanks again for flagging!